packaging/hudi-flink-bundle/pom.xml

```diff
@@ -106,6 +106,7 @@
                   <include>com.fasterxml.jackson.core:jackson-annotations</include>
                   <include>com.fasterxml.jackson.core:jackson-databind</include>
                   <include>com.fasterxml.jackson.core:jackson-core</include>
+                  <include>com.fasterxml.jackson.datatype:jackson-datatype-jsr310</include>
```

Comment:
I think we'd need to add this one to Spark bundle as well

Reply:
Let's also enable Column Stats (write-path) in our Integration tests as well to make sure we can capture these